fbw-a32nx/src/systems/instruments/src/SD/PagesContainer.tsx

```diff
@@ -42,219 +42,237 @@ const FCTL_CONDITION_TIMER_DURATION = 20;
+      setStsPressedTimer(STS_DISPLAY_TIMER_DURATION);
+    }
+  };
 
-    const updateCallback = (deltaTime) => {
-        if (ecamAllButtonPushed && !prevEcamAllButtonState) { // button press
-            setPage((prev) => (prev + 1) % 12);
-            setCurrentPage((prev) => (prev + 1) % 12);
-            setEcamCycleInterval(setInterval(() => {
-                setCurrentPage((prev) => {
-                    setPage((prev + 1) % 12);
-                    return (prev + 1) % 12;
-                });
-            }, 1000) as unknown as number);
-        } else if (!ecamAllButtonPushed && prevEcamAllButtonState) { // button release
-            clearInterval(ecamCycleInterval);
-        } else if (!ecamAllButtonPushed) {
-            if (currentPage !== SdPages.Status) {
-                setStsPrevPage(currentPage);
-            }
-            const newPage = page;
-            if (newPage !== -1) {
-                setCurrentPage(newPage);
-            } else {
-                setCurrentPage(pageWhenUnselected);
-            }
+  const updateCallback = (deltaTime) => {
+    if (ecamAllButtonPushed && !prevEcamAllButtonState) {
+      // button press
+      setPage((prev) => (prev + 1) % 12);
+      setCurrentPage((prev) => (prev + 1) % 12);
+      setEcamCycleInterval(
+        setInterval(() => {
+          setCurrentPage((prev) => {
+            setPage((prev + 1) % 12);
+            return (prev + 1) % 12;
+          });
+        }, 1000) as unknown as number,
+      );
+    } else if (!ecamAllButtonPushed && prevEcamAllButtonState) {
+      // button release
+      clearInterval(ecamCycleInterval);
+    } else if (!ecamAllButtonPushed) {
+      if (currentPage !== SdPages.Status) {
+        setStsPrevPage(currentPage);
+      }
+      const newPage = page;
+      if (newPage !== -1) {
+        setCurrentPage(newPage);
+      } else {
+        setCurrentPage(pageWhenUnselected);
+      }
 
-            switch (fwcFlightPhase) {
-            case 10:
-            case 1:
-                setCrzCondTimer(CRZ_CONDITION_TIMER_DURATION);
-                setPageWhenUnselected(SdPages.Door);
-                // TODO: Emergency Generator Test displays ELEC
-                // Needs system implementation (see A320_NEO_INTERIOR Component ID EMER_ELEC_PWR [LVar: L:A32NX_EMERELECPWR_GEN_TEST])
-                checkApuPage(deltaTime);
-                checkEnginePage(deltaTime);
-                break;
-            case 2:
-                const sidestickPosX = SimVar.GetSimVarValue('L:A32NX_SIDESTICK_POSITION_X', 'Number');
-                const sidestickPosY = SimVar.GetSimVarValue('L:A32NX_SIDESTICK_POSITION_Y', 'Number');
-                const rudderPos = SimVar.GetSimVarValue('RUDDER PEDAL POSITION', 'Position');
-                const controlsMoved = Math.abs(sidestickPosX) > 0.05 || Math.abs(sidestickPosY) > 0.05 || Math.abs(rudderPos) > 0.2;
+      switch (fwcFlightPhase) {
+        case 10:
+        case 1:
```

Comment:
What are the rules for adding brackets {} on switch statements? Seems a bit inconsistent to only wrap a case in them if you have a variable involved.

Reply:
I think it's to do with another linting rule about not declaring variables in switch case